#!/bin/sh /usr/share/dpatch/dpatch-run
## 04-localconf-kmk.dpatch by Patrick Winnertz <patrick.winnertz@skolelinux.org>
## expanded my Michael Meskes <meskes@debian.org> to set paths
##
## DP: Don't build the module for testing and don't build any guest additions.
## DP: both depends on kernel headers, which are not available atm.

@DPATCH@

diff -Naurp virtualbox-ose-1.6.0-dfsg.orig/LocalConfig.kmk virtualbox-ose-1.6.0-dfsg/LocalConfig.kmk
--- virtualbox-ose-1.6.0-dfsg.orig/LocalConfig.kmk	1970-01-01 00:00:00.000000000 +0000
+++ virtualbox-ose-1.6.0-dfsg/LocalConfig.kmk	2008-05-09 09:25:16.000000000 +0000
@@ -0,0 +1,12 @@
+VBOX_WITH_ADDITION_DRIVERS =
+VBOX_ADDITIONS_LINUX_ONLY =
+VBOX_WITH_INSTALLER = 1
+VBOX_WITH_LINUX_ADDITIONS = 1
+VBOX_WITH_TESTCASES =
+VBOX_WITH_TESTSUITE =
+VBOX_WITH_USB =
+VBOX_WITH_REGISTRATION_REQUEST =
+DEFS += RTPATH_APP_DOCS=\"/usr/share/doc/virtualbox\"
+DEFS += RTPATH_APP_PRIVATE_ARCH=\"/usr/lib/virtualbox\"
+DEFS += RTPATH_APP_PRIVATE=\"/usr/share/virtualbox\"
+DEFS += RTPATH_SHARED_LIBS=\"/usr/lib/virtualbox\"
